Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3.18
+
+ENV TFC_OPS_VER="3.5.1"
+ENV TFC_OPS_DISTRIB="tfc-ops_${TFC_OPS_VER}_Linux_x86_64.tar.gz"
+
+COPY ./dump-and-save.sh  /usr/local/bin/dump-and-save.sh
+COPY ./tfc-dump.pl       /usr/local/bin/tfc-dump.pl
+
+# Install tfc-ops, 1Password CLI, perl, and jq
+RUN chmod +x /usr/local/bin/dump-and-save.sh /usr/local/bin/tfc-dump.pl \
+ && cd /tmp \
+ && wget https://github.com/silinternational/tfc-ops/releases/download/v${TFC_OPS_VER}/${TFC_OPS_DISTRIB} \
+ && tar zxf ${TFC_OPS_DISTRIB} \
+ && rm LICENSE README.md ${TFC_OPS_DISTRIB} \
+ && mv tfc-ops /usr/local/bin \
+ && chmod +x /usr/local/bin/tfc-ops \
```

Comment:
likewise here, zip should retain the execute bit so the `chmod` isn't strictly necessary I believe

Reply:
Agreed.  Still a habit.